DFC.ServiceTaxonomy.ContentPickerPreview/wwwroot/Scripts/vue-multiselect-wrapper-preview.js

```diff
@@ -82,9 +84,28 @@ function initVueMultiselectPreview(element) {
                         url : '/Contents/ContentItems/' + contentItemId,
                         type: 'GET',
 
-                        success: function(data){
-                            $('#previewhere').html(data);
+                        success: function (data) {
+                            var temp = $('<temp>').append($.parseHTML(data));
+                            var occupationLabels = $('#OccupationLabels', temp);
```

Comment:
The preview editor for the content picker should be generic, so that we can re-use it.